docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,198 @@
+
+We go with the approach of having one Elasticsearch per region and configuring Zeebe to export the Elasticsearch data in both regions.
+
+#### WebApps
+
+WebApps are an **active-passive** component and in its current state also not highly available since we're limited by the included exporters that would cause data issues when running multiple instances at the same time.
+
+In every region, there can be only a single WebApp instance running (i.e. in Region 0, 1 instance of Operate, 1 instance of Tasklist; same in Region 1)
+
+This means that one instance will be actively serving traffic, while the other one is on standby.
+
+##### Operate
+
+In the event of a total active region loss, the following data will be lost:
+
+- Batch Operations <!-- Do we have a docs link for this? -->
+
+##### Tasklist
+
+In the event of a total active region loss, the following data will be lost:
+
+- Assignments of tasks
+
+## Requirements
+
+- Camunda 8
+  - Minimum [Helm chart version](https://github.com/camunda/camunda-platform-helm) **9.3+**
+  - Minimum component images
+    - Elasticsearch **8.9+**
+    - Operate **8.5+**
+    - Tasklist **8.5+**
+    - Zeebe **8.5+**
+    - Zeebe Gateway **8.5+**
+- Two Kubernetes clusters
+  - OpenShift is not supported
+  - those need to be able to contact each other via, for example, VPC peering or similar means
+  - an [example implementation](<!-- TODO: Link -->) of two VPC peered Kubernetes clusters based on AWS EKS
+- Elasticsearch usage
+  - OpenSearch (both managed and self-managed) is not supported
+- Maximum RTT (round trip time) of 100 ms between the two Kubernetes clusters
+- Open Ports between the two Kubernetes clusters
+  - **9200** for Elasticsearch since Zeebe has to push data cross-region
+  - **26500** for the Zeebe gateway from client/worker communication
+  - **26501** for the Zeebe brokers and Zeebe gateway communication
+  - **26502** for the Zeebe brokers and Zeebe gateway communication
+- Only specific combinations of Zeebe broker counts and replication factors are supported
+  - `clusterSize` must be a multiple of **two** and a minimum of **four** to evenly distribute the brokers across the two regions
+  - `replicationFactor` must be **four** to ensure that the partitions are evenly distributed across the two regions
+  - `partitionCount` is not restricted and depends on your workload requirements, consider having a look at [understanding sizing and scalability behavior](../../../components/best-practices/architecture/sizing-your-environment.md#understanding-sizing-and-scalability-behavior)
+  - for further information and visualization of the partition distribution consider conducting the documentation about [partitions](../../../components/zeebe/technical-concepts/partitions.md)
+
+## Limitations
+
+- Camunda 8 must be installed with the [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm)
```

Comment:
maybe an internal docs link is better: http://localhost:3000/docs/next/self-managed/platform-deployment/helm-kubernetes/overview/#use-helm-to-install-on-kubernetes